File: app/src/main/java/com/liushu/crazyandroid/ui/stage01/chapter02/activity/Demo020201Activity.java
package com.liushu.crazyandroid.ui.stage01.chapter02.activity;

import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import com.jaydenxiao.common.base.BaseActivity;
import com.liushu.crazyandroid.R;

import butterknife.BindView;

public class Demo020201Activity extends BaseActivity {

    @BindView(R.id.iv_back)
    ImageView mIvBack;
    @BindView(R.id.tv_title_name)
    TextView mTvTitleName;

    @Override
    public int getLayoutId() {
        return R.layout.activity_demo020201;
    }


    @Override
    public void initView() {
        mTvTitleName.setText("丰富的表格布局");
        mIvBack.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                finish();
            }
        });
    }
}

File: app/src/main/java/com/liushu/crazyandroid/ui/stage01/chapter02/activity/Demo020201Activity.kt
package com.liushu.crazyandroid.ui.stage01.chapter02.activity

import com.jaydenxiao.common.base.BaseActivity
import com.liushu.crazyandroid.R
import kotlinx.android.synthetic.main.head_title_layout.*

class Demo020201Activity : BaseActivity() {


    override val layoutId: Int
        get() = R.layout.activity_demo020201


    override fun initView() {
        tv_title_name!!.text = "丰富的表格布局"
        iv_back!!.setOnClickListener { finish() }
    }
}
